subsys/bluetooth/host/adv.c

```diff
@@ -1930,7 +1930,7 @@ void bt_hci_le_adv_set_terminated(struct net_buf *buf)
 	struct bt_hci_evt_le_adv_set_terminated *evt;
 	struct bt_le_ext_adv *adv;
 	uint16_t conn_handle;
-#if (CONFIG_BT_ID_MAX > 1) && (CONFIG_BT_EXT_ADV_MAX_ADV_SET > 1)
+#if (CONFIG_BT_MAX_CONN > 0) && (CONFIG_BT_EXT_ADV_MAX_ADV_SET > 1)
```

Comment:
`BT_MAX_CONN` has a specified range with a minimum value of `1`, so testing for `> 0` will always be true if the macro is defined. Is it always guaranteed to be defined here, i.e. is `CONFIG_BT_CONN` always enabled in this part of the code? If yes, then just drop the first condition the if statement. If no, then the condition should read `#if defined(CONFIG_BT_CONN) && ...`.